readthedocs/proxito/README.rst

```diff
@@ -42,6 +42,80 @@ path prefix:
                                     version     |
                                              filename
 
+URL patterns
+------------
+
+URL patterns define how docs are served, and how the path is parsed by proxito.
+
+For example, the default paths used are:
+
+- Multi version project: ``/<language>/<version>/<filename>``
+- Single version project: ``/<filename>``
+- Subproject: ``/projects/<subproject-alias>/<filename>``
+
+URL patterns can be used to change these paths per project, with the following attributes:
+
+urlpattern:
+   URL pattern to change the path for multi version and single version projects.
+
+urlpattern_subproject:
+   URL pattern to change the path up to the subproject alias (the ``/projects`` prefix).
+   To change the pattern of the subproject itself, use ``urlpattern``.
```

Comment:
Is this what we're saying? Or will the parent projects urlpattern apply to all subprojects? 

```suggestion
   To change the pattern of the subproject itself, use ``urlpattern`` on the subproject.
```

Reply:
your suggestion is correct